src/main/java/co/rsk/federate/FederationProvider.java

```diff
@@ -39,6 +39,8 @@ public interface FederationProvider {
     // The currently "retiring" federation's address
     Optional<Address> getRetiringFederationAddress();
 
+    // The currently "proposed" federation
```

Comment:
Do we really need all these simple comments on top of the method definition? There´s no logic here. I think they are not adding much value really.

Reply:
I agree, I have made better javadocs for the interface. Let me know what you think